android/src/main/java/com/bitmovin/player/reactnative/RNPlayerViewManager.kt

```diff
@@ -255,6 +259,12 @@ class RNPlayerViewManager(private val context: ReactApplicationContext) : Simple
             val pictureInPictureHandler = view.pictureInPictureHandler ?: RNPictureInPictureHandler(context)
             view.pictureInPictureHandler = pictureInPictureHandler
             view.pictureInPictureHandler?.isPictureInPictureEnabled = isPictureInPictureEnabled
+
+            val rnPlayerConfigWrapper = view.config
+                ?: playerConfig?.let { JsonConverter.toRNPlayerViewConfigWrapper(it) }
+            val rnStyleConfigWrapper = playerConfig?.let { JsonConverter.toRNStyleConfigWrapper(it) }
+            val nativePlayerViewConfig = rnPlayerConfigWrapper?.playerViewConfig ?: PlayerViewConfig()
```

Comment:
I understand what you are doing, but I think it is a bit confusing / difficult to see.

A `PlayerConfig` can't be a `PlayerViewConfig`, but the code here suggests that we are doing this "cast" here. And it indeed works, because you added in that case in the `toRNPlayerViewConfigWrapper` function, but as said I think that's quite confusing.
I'll reach out to you in a place we can have a faster paced discussion.